### ISSUE TITLE
VS2015 build issue

### ISSUE THREAD

**Issue by michalo2882** [Microsoft/DirectXShaderCompiler#347]
Hi, I observed VS 2015 build issue with recent commits.
Issue is gone when I disable HLSL_INCLUDE_TESTS
Here's error:

```
D:\DXCompiler\tools\clang\unittests\HLSL\ExecutionTest.cpp(2596): error C2440: 'initializing': cannot convert from 'initializer list' to 'TableParameter' [D:\DXCompiler\.bin_x64\tools\clang\unittests\HLSL\clang-hlsl-tests.vcxproj]
  D:\DXCompiler\tools\clang\unittests\HLSL\ExecutionTest.cpp(2596): note: No constructor could take the source type, or constructor overload resolution was ambiguous
```

**Comment by youngkim93** [comment]
Hi, 
This problem should have been resolved with #348. Please let us know if you are still having this issue with this payload. Thanks!

**Comment by marcelolr** [comment]
@michalo2882, please reopen or file a new issue if you're still running into problems with your setup - thanks!
